mms/go/cmd/gocqlxapp/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"goapp/internal/log"
+	"goapp/internal/scylla"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx"
+	"github.com/scylladb/gocqlx/qb"
+	"github.com/scylladb/gocqlx/table"
+	"go.uber.org/zap"
```

Comment:
Another logger?

Reply:
hmm... Let me check.

Reply:
Zap is the logger. The other import just defines a helper method to create the otherwise quite verbose setup of the zap logger.

Perhaps we should use golog?